pkgs/tools/graphics/metapixel/default.nix

```
@@ -0,0 +1,33 @@
+{stdenv, fetchFromGitHub, libpng, libjpeg, giflib, perl, pkg-config}:
+
+stdenv.mkDerivation {
+  pname = "metapixel";
+  version = "1.0.2";
+
+  src = fetchFromGitHub {
+    owner = "schani";
+    repo = "metapixel";
+    rev = "98ee9daa093b6c334941242e63f90b1c2876eb4f";
+    fetchSubmodules = true;
+    sha256 = "0r7n3a6bvcxkbpda4mwmrpicii09iql5z69nkjqygkwxw7ny3309";
+  };
+
+  makeFlags = [ "PREFIX=build" "metapixel" ];
```

The following should work and you should be able to drop the `installPhase` too with this change:

```suggestion
  makeFlags = [ "PREFIX=$(out)" "metapixel" ];
```

---

I think I actually have to just drop the PREFIX part. the compile step does not use it, and the install script does the funky man page stuff. Nice catch.